src/main/kotlin/calculator/Delimiters.kt

```diff
@@ -0,0 +1,39 @@
+package calculator
+
+class Delimiters {
+
+    private val _delimiters: MutableList<String> = mutableListOf()
+    val values: List<String>
+        get() = _delimiters.toList()
+
+    fun add(delimiter: String) {
+        _delimiters.add(delimiter)
+    }
+
+    fun add(delimiterList: List<String>) {
+        _delimiters.addAll(delimiterList)
+    }
+
+    fun validateInput(input: String) {
+        val regex = _delimiters.joinToString("|").toRegex()
+        
+        input.split(regex).forEach { token ->
+            validateIsNumeric(token)
+        }
+    }
+
+    private fun validateIsNumeric(token: String) {
+        if (token.toIntOrNull() == null) {
+            throw RuntimeException("입력 값은 쉼표(,), 콜론(:), 커스텀 구분자를 제외한 나머지 문자는 모두 숫자로 입력해야 합니다.")
```

Comment:
token 값이 무엇인지도 예외 메세지에 포함되어있으면 조금 더 에러를 파악하기 쉽겠죠? 🤗

```suggestion
            throw RuntimeException("입력 값은 쉼표(,), 콜론(:), 커스텀 구분자를 제외한 나머지 문자는 모두 숫자로 입력해야 합니다. token: $token")
```